### ISSUE TITLE
RuntimeException: Unable to stop service com.hbisoft.hbrecorder.ScreenRecordService

### ISSUE THREAD

**Issue by aruproy** [HBiSoft/HBRecorder#44]
ScreenRecorderService crashed in production environment. Hope this crash can be avoided with a simple NULL check. I'd appreciate any help. 

**Describe the bug**
ScreenRecorderService within the HBRecorder crashed on `hbRecorder.stopScreenRecording()` 

**Log**
```
uncaughtException:
Unable to stop service com.hbisoft.hbrecorder.ScreenRecordService@bf716e7: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.media.projection.MediaProjection.stop()' on a null object reference

StackTrace:
java.lang.RuntimeException: Unable to stop service com.hbisoft.hbrecorder.ScreenRecordService@bf716e7: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.media.projection.MediaProjection.stop()' on a null object reference

StackTraceDetails:
java.lang.RuntimeException: Unable to stop service com.hbisoft.hbrecorder.ScreenRecordService@bf716e7: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.media.projection.MediaProjection.stop()' on a null object reference
        at android.app.ActivityThread.handleStopService(ActivityThread.java:4583)
        at android.app.ActivityThread.access$2100(ActivityThread.java:274)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2139)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:237)
        at android.app.ActivityThread.main(ActivityThread.java:8167)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:496)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1100)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.media.projection.MediaProjection.stop()' on a null object reference
        at com.hbisoft.hbrecorder.ScreenRecordService.f(SourceFile:7)
        at com.hbisoft.hbrecorder.ScreenRecordService.onDestroy(SourceFile:2)
        at android.app.ActivityThread.handleStopService(ActivityThread.java:4563)

```
**Can it be reproduced in demo app**
This crash is reported in production environment.

**HBRecorder version**
HBRecorder:0.1.14

**Device information**
Samsung SM-A715F
SDK version 29

**Comment by aruproy** [comment]
@HBiSoft Please resolve this issue.

**Comment by HBiSoft** [comment]
This is fixed in `0.1.15`
